bookstore-scrapper/algolia/updateAlgoliaIndex.ts

```diff
@@ -20,7 +26,17 @@ const searchableAttributes: string[] = [
   "entry.isbn",
 ];
 
-const rankings: string[] = ["exact", "words", "proximity"];
+const rankings: string[] = [
+  "asc(entry.code)",
+  "asc(entry.dept.abbreviation)",
+  "asc(entry.dept.name)",
```

Comment:
Curious why we want these to be ascending instead of best match?

Reply:
This is related to sorting entries rather than ranking them.

Results will be received with courses sorted in ascending alphabetical order placed first (ranked according to best matched) followed by books that match the query.

For example, searching for BIO would provide you with all the best matched biology courses sorted in alphabetical order placed first (ex. BIO 1A, BIO 1B). Then, there would be books with the letters "BIO" ranked according to best match.